test/Test/Database/LSMTree/StateMachine.hs

```diff
@@ -1273,10 +1284,18 @@ runIOSim action lookUp = ReaderT $ \ !env -> do
           Class.mupserts (unwrapTable $ lookUp' tableVar) kmups
         RetrieveBlobs blobRefsVar -> catchErr handlers $
           fmap WrapBlob <$> Class.retrieveBlobs (Proxy @h) session (unwrapBlobRef <$> lookUp' blobRefsVar)
-        CreateSnapshot merrs label name tableVar ->
-          runRealWithInjectedErrors "CreateSnapshot" env merrs
-            (Class.createSnapshot label name (unwrapTable $ lookUp' tableVar))
-            (\() -> Class.deleteSnapshot session name)
+        CreateSnapshot mcorrsOrErrs label name tableVar ->
```

Comment:
The generator for `CreateSnapshot` generates `Nothing` by default, so we're currently not running tests with coruption at all

Reply:
I see you fixed this.